src/Revise.jl

```diff
@@ -1158,9 +1166,15 @@ function __init__()
     mode = get(ENV, "JULIA_REVISE", "auto")
     if mode == "auto"
         if isdefined(Base, :active_repl_backend)
-            steal_repl_backend(Base.active_repl_backend::REPL.REPLBackend)
+            if VERSION >= v"1.5.0-DEV.282"
+                pushfirst!(Base.active_repl_backend.ast_transforms, revise_first)
+            else
+                steal_repl_backend(Base.active_repl_backend::REPL.REPLBackend)
+            end
         elseif isdefined(Main, :IJulia)
             Main.IJulia.push_preexecute_hook(revise)
```

Comment:
(Eventually IJulia will just call the `REPL.repl_ast_transforms`, so you won't need this special case.)